### DISCUSSION TITLE
Meaning of this text

### DISCUSSION THREAD

**Discussion by JA16122000** [mrlt8/docker-wyze-bridge#573] [Q&A]
Hi!

I am seeing an error?

 2022/10/24 15:13:59 [WyzeBridge] WARNING: unclosed <socket.socket fd=5, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('0.0.0.0', 0)>

What does that mean, and is this an issue?

**Comment by mrlt8** [comment]
Hmm, I haven't seen that before. Is this on startup only?

**Reply by JA16122000** [reply]
Correct, only at startup.
Everthing is working like a charm, so i don't think it is an issue :)
I was just wondering
